code/parse/lua.cpp

```diff
@@ -12713,6 +12714,21 @@ ADE_VIRTVAR(MouseControlStatus, l_Mouse, "boolean", "Gets and sets the retail mo
 		return ADE_RETURN_FALSE;
 }
 
+ADE_FUNC(getMouseSensitivity, l_Mouse, NULL, "Gets mouse sensitivity setting", "number", "Mouse sensitivity in range of 0-9")
+{
+	return ade_set_args(L, "i", Mouse_sensitivity);
+}
+
+ADE_FUNC(getJoySensitivity, l_Mouse, NULL, "Gets joystick sensitivity setting", "number", "Joystick sensitivity in range of 0-9")
+{
+	return ade_set_args(L, "i", Joy_sensitivity);
+}
+
+ADE_FUNC(getJoyDeadzone, l_Mouse, NULL, "Gets joystick deadzone setting", "number", "Joystick deadzone in range of 0-45")
```

Comment:
It's just an old internal name for the io/controls library:
ade_lib l_Mouse("Controls", NULL, "io", "Controls library");

It's not really specific to the mouse anymore as it also contains TrackIR polling functions and is the logical place for joystick-related ones too.